core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -25,9 +25,13 @@ function getThreadDumpEnabled() {
     return threadDumpEnabled;
 }
 
-function formatStatus(status, type) {
+function formatStatus(status, type, row) {
```

Comment:
can you move the

```js
if (row.isBlacklisted) return "Blacklisted";
```
into this function as well?  no reason for it to be separated.

Reply:
Ok, done.